.slashrc

```diff
@@ -262,6 +264,11 @@ class MediaPlugin(slash.plugins.PluginInterface):
     filename = os.path.join(
       slash.context.session.results.global_result.get_log_dir(), "results.xml")
     tree.write(filename)
+  def _get_call_timeout(self):
```

Comment:
Please move this towards the top of the class where other functions that start with `_` are located.  Also, please make sure you have an empty line between function definitions.

Reply:
moved _get_call_timeout to top where other _ are defined